eng/build.ps1

```diff
@@ -14,6 +14,8 @@ Param(
   [Parameter(ValueFromRemainingArguments=$true)][String[]]$properties
 )
 
+. $PSScriptRoot\common\tools.ps1
```

Comment:
Should we include this only if $vs == true?

Reply:
Yeah, I did that locally to fix the build break.  I'm a powershell noob, so I'm not exactly sure what I did wrong here.  Regardless moving it seems to fix things.